test/ssr/ssr-string.spec.js

```diff
@@ -360,6 +358,39 @@ describe('SSR: renderToString', () => {
     })
   })
 
+  it('custom directives', done => {
+    const custom = jasmine.createSpy()
```

Comment:
We should implement some mock behavior of manipulating the VNode and assert the result. (see https://github.com/vuejs/vue/tree/next/src/platforms/web/server/directives)